packages/components/src/components/domainproperties/issues/IssuesPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,221 @@
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.name === null ? '' : model.name;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        placeholder="Enter a singular name for this Issue"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class PluralItemNameInput extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.name === null ? '' : model.name;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Plural items name" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="pluralItemName"
+                        type="text"
+                        placeholder="Enter a plural name for this Issue"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class CommentSortDirectionDropDown extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.commentSortDirection;
+
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Comment sort direction" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        componentClass="select"
+                        id="commentSortDirection"
+                        onChange={onInputChange}
+                        value={value ? value : ''}
+                    >
+                        <option key="Oldest first" value="ASC">
+                            Oldest first
+                        </option>
+                        <option key="Newest first" value="DESC">
+                            Newest first
+                        </option>
+                    </FormControl>
```

Comment:
For consistency, we are trying to move away from using the FormControl for select inputs and instead use the ui-component `<SelectInput>`. Looks like you have some other usages of that component in this PR, so that's good.

Reply:
Now changed to use SelectInput